Arcade/TNT Floor/map.xml

```diff
@@ -0,0 +1,46 @@
+<map proto="1.4.0">
+<name>TNT Floor</name>
+<version>1.0.0</version>
+<objective>Shoot out the TNT under the other team to win!</objective>
+<gamemode>arcade</gamemode>
+<authors>
+    <authors> <author uuid="d1873b18-8194-429b-b6d5-eb7eb897071d"/> <!-- AVVERSION --> </authors>
+</authors>
+<contributors>
+    <contributor uuid="a9f11454-beac-4cc3-a138-1780802c8794" contribution="XML"/> <!-- Crazy_ --> </contributors>
+</contributors>
+<teams>
+    <team id="red" color="dark red" max="3">Red</team>
+    <team id="blue" color="blue" max="3">Blue</team>
+</teams>
+<kits>
+    <kit id="play">
+        <item slot="0" enchantment="arrow infinite;flame">bow</item>
+        <item slot="28">arrow</item>
+        <gamemode>adventure</gamemode>
+        <gamemode>blitz</gamemode>
+    </kit>
+</kits>
+<spawns>
+    <spawn team="red" kit="play" yaw="135">
+        <region><point>-138.5,116,-784.5</point></region>
+    </spawn>
+    <spawn team="blue" kit="play" yaw="-45">
+        <region><point>-159.5,116,-805.5</point></region>
+    </spawn>
+    <default>
+        <region><point>-148,117,-794</point></region>

```

Comment:
When I tested the map in-game, I noticed the spectator spawn were off-centered. Make sure you end your x and z coordinates in `.5` if you want players to be centered on that block. I've also spaced out and indented this properly so it looks cleaner.
```suggestion
        <region>
            <point>-148.5,117,-794.5</point>
        </region>
```